app/src/main/java/Activities/AddRecordActivity.java

```diff
@@ -173,6 +174,15 @@ public void onViewPhotos(View v){
         startActivity(intent);
     }
 
+    //Set Geolocation click
+    public void onSetGeolocationClick(View v){
+
```

Comment:
this.onCurrentPage = 0;

Photos are not added to database if the above line is not there. I can't test it if that work right now since I'm having trouble connecting to elastic search database, could you try adding it and see if the photos are saved properly?